### PR TITLE
Code changes from repo rename

### PR DESCRIPTION
**Pull request by DerekRoberts**
---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://quickstart-openshift-1092-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://quickstart-openshift-1092-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/quickstart-openshift/actions/workflows/merge-main.yml)

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nr-quickstart-typescript",
+  "name": "quickstart-openshift",
```
Comment (mishraomp) [inline backend/package.json:2]: should we update this one to `quickstart-openshift-backend` in this one, or may be a separate PR?

**Review by mishraomp — APPROVED** [review]
LGTM

File: backend/package.json (hunk 1 above)
Comment (DerekRoberts) [reply backend/package.json:2]: If we're not publishing it shouldn't matter.  I renamed to `backend`, which is appropriately generic.
